components/prism-keepalived.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.keepalived = {
+	'comment': {
+		pattern: /(^|[^"{\\])#.*/,
+		lookbehind: true
+	},
+	'property': {
+		pattern: /([\s\t]*)(?:\bglobal_defs|notification_email|notification_email_from|smtp_server|smtp_connect_timeout|router_id|virtual_server|fwmark|delay_loop|lb_algo|lb_kind|persistence_timeout|persistence_granularity|virtualhost|protocol|sorry_server|real_server|weight|TCP_CHECK|MISC_CHECK|misc_path|HTTP_GET|SSL_GET|url|path|digest|connect_port|connect_timeout|retry|delay_before_retry|vrrp_instance|state|interface|mcast_src_ip|lvs_sync_daemon_inteface|virtual_router_id|priority|advert_int|smtp_alert|authentication|auth_type|auth_pass|virtual_ipaddress|virtual_ipaddress_excluded|notify_master|notify_backup|notify_fault|vrrp_sync_group|vrrp_script|interval|script)\b/,
```

Comment:
1. The lookbehind is unnecessary.
2. I think this pattern is supposed to be of the form `\b(?:foo|...|bar)\b`, isn't it?
3. Please sort the list.

Reply:
> The lookbehind is unnecessary.

The `comment` token is copied from nginx.

If I remove `(^|[^"{\\])` from `/(^|[^"{\\])#.*/`, it can match `"it is a # string"`, so I put the `string` token before it, can pass the test and above example.

> I think this pattern is supposed to be of the form \b(?:foo|...|bar)\b, isn't it?

In fact, when I use nginx, I found that `\b(?:index|types|...)\b` can match `index.html` and `/etc/nginx/mime.types`, this result is not what we except, so I try to find a solution to avoid it. This is just a temporary practice, I have not thought of a better solution.

> Please sort the list.

The order of the list is specified in the document. In the [old document](https://www.keepalived.org/doc/configuration_synopsis.html), there is a table describing all the attributes. I also just know that the new document is updated in another interface and this list will be updated. Do you think it is better to use the order of appearance in the document (now), or alphabetical sorting?

Reply:
>  I found that \b(?:index|types|...)\b can match index.html and /etc/nginx/mime.types

You already have a pattern that matches all paths. Can't we just match paths before this pattern?

> Do you think it is better to use the order of appearance in the document (now), or alphabetical sorting?

Alphabetical sorting. It's easy to see if certain words are included and it's obvious where you have to add new ones (if new ones were added).

Reply:
The list is so long that could I use a string array to build it? like [factor](https://github.com/PrismJS/prism/blob/master/components/prism-factor.js#L381-L399).

Reply:
In the case of factor, I think it's justified because the combinations there contain a lot of special regex characters that have to be escaped. That's not the case here, so I don't see the point in this, so I think that the overhead of a function is a little too much.

If you just want to do something like `pattern: RegExp('\\b' + ['word1', 'word2'].join('|') + '\\b')`, then sure! Go for it!